frontend/src/scenes/persons/PersonPreview.tsx

```diff
@@ -39,9 +36,9 @@ export function PersonPreview(props: PersonPreviewProps): JSX.Element {
         <div className="flex flex-col overflow-hidden max-h-80 max-w-160 gap-2">
             <div className="flex items-center justify-between">
```

Comment:
Some breathing room:
```suggestion
            <div className="flex items-center justify-between min-h-10 px-2">
```